proparse/src/main/java/org/prorefactor/core/nodetypes/ProgramRootNode.java

```diff
@@ -37,13 +33,4 @@ public int getSubtypeIndex() {
     return 6;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 6 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS109)